rezerwacje-front/src/views/Schedule/components/CoursesSelection/CoursesSelection.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
+import { makeStyles } from '@material-ui/styles';
+import PropTypes from 'prop-types';
+import { 
+  Card, 
+  CardContent,
+  Grid, 
+  Typography, 
+  FormControl,
+  Select,
+  InputLabel,
+  Chip,
+  Input,
+  MenuItem,
+} from '@material-ui/core';
+
+const useStyles = makeStyles(theme => ({
+  formControl: {
+    width: '100%',
+  },
+  chips: {
+    display: 'flex',
+    flexWrap: 'wrap',
+  },
+  chip: {
+    margin: 2,
+  },
+}));
+
+const CoursesSelection = props => {
+  const { className, ...rest } = props;
+
+  const classes = useStyles();
+
+  const courses = props.courses;
+
+  const [selectedCourses, setSelectedCourses] = useState(courses.slice(0));
+
+  const handleSelectedCoursesChange = event => {
+    const value = event.target.value
+    setSelectedCourses(value)
+    props.updateCourses(value)
+  };
```

Comment:
It seems there's the same problem as with rooms. I guess rooms and selectedRooms should be distinguished in props? E.g. for cycles you have props.cycles and props.cycle for selected cycle